docs/installing_deis/linode.rst

```diff
@@ -0,0 +1,218 @@
+:title: Installing Deis on Linode
+:description: How to provision a multi-node Deis cluster on Linode
+
+.. _deis_on_linode:
+
+Linode
+======
+
+In this tutorial, we will show you how to set up your own 3-node cluster on Linode.
+
+Please :ref:`get the source <get_the_source>` and refer to the scripts in `contrib/linode`_
+while following this documentation.
+
+.. important::
+
+    Linode support is untested by the Deis team, so we rely on the community to
+    improve this documentation and fix bugs. We greatly appreciate the help!
+
+
+Prerequisites
+-------------
+
+Before we can begin to provision a cluster on Linode, let's get a few things squared away.
+
+
+Enable KVM Hypervisor
+^^^^^^^^^^^^^^^^^^^^^
+
+Navigate to the `Linode Account Settings`_ page and change the Hypervisor Preference to ``KVM``.
+
+Although it is possible to provision CoreOS under Xen on Linode it is much more difficult and
+the tools included only work with the KVM Hypervisor.
+
+
+Obtain Linode API Key
+^^^^^^^^^^^^^^^^^^^^^
+
+Next, navigate to the `Linode API Keys`_ page and generate an API key. Take note of the key,
+as you will need it later when you provision your cluster.
+
+
+Install Python and Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The scripts used to provision the cluster are written for Python 2.7 and require a few
+dependencies be installed with the `pip`_ package manager. If you are on OS X or Linux you
+likely have these already available.
+
+Lets install our dependencies:
+
+.. code-block:: console
+
+    $ pip install -r contrib/linode/requirements.txt
+
+
+Generate SSH Key
+^^^^^^^^^^^^^^^^
+
+If you don't already have a SSH key, the following command will generate
+a new keypair named "deis":
+
+.. code-block:: console
+
+    $ ssh-keygen -q -t rsa -f ~/.ssh/deis -N '' -C deis
+
+
+Check System Requirements
+-------------------------
+
+Please refer to :ref:`system-requirements` for resource considerations when choosing a Linode
+plan to run Deis. The smallest recommended cluster consists of 3 Linode 4096s.
```

Comment:
Could you replace with or add this wording? We just added this to the other "Installing" docs for consistency:
```
A Deis cluster must have 3 or more nodes. See :ref:`cluster-size` for more details.
```